src/components/CreateInstructorPage/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+import { CreateInstructorForm } from './CreateInstructorForm';
+import StatusAlert from '../StatusAlert';
+import LoadingSpinner from '../LoadingSpinner';
+
+class CreateInstructorPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      startedFetching: false,
+    };
+    this.handleInstructorCreate = this.handleInstructorCreate.bind(this);
+    this.setStartedFetching = this.setStartedFetching.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.fetchInstructorOptions();
+    this.setStartedFetching();
+  }
+
+  setStartedFetching() {
+    this.setState({ startedFetching: true });
+  }
+
+  processSocialLinks(links) {
+    if (!links) {
+      return [];
+    }
+
+    return links.map(link => (
+      ({
+        id: null,
+        type: link.type,
+        title: link.title,
```

Comment:
In my test, the `title` field was not being passed along to the endpoint. The others were though... Do you see the same thing?

Reply:
In my test just now social link `title` _is_ getting sent